doc/nrf/known_issues.rst

```diff
@@ -1178,6 +1178,21 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-7-0
+
+DRGN-16317: The SoftDevice Controller may discard LE Extended Advertising Reports
+
+If there is insufficient memory available or the Host is not able to process HCI events in time, the SoftDevice Controller may discard LE Extended Advertising Reports.
+If advertising data is split across multiple reports and any of these are discarded, the Host will not be able to reassemble the data.
+
+**Workaround:** Increase :kconfig:`CONFIG_BT_BUF_EVT_RX_COUNT` until events are no longer discarded.
+
+.. rst-class:: v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+DRGN-15903: BT_CTLR_TX_PWR is ignored by the SoftDevice Controller
+
+  **Workaround:** Use the HCI command Zephyr Write Tx Power Level to dynamically set TX power.
```

Comment:
```suggestion
DRGN-15903: ``BT_CTLR_TX_PWR`` is ignored by the SoftDevice Controller

  **Workaround:** Use the HCI command Zephyr Write Tx Power Level to dynamically set TX power.
```
This needs an indented description line under the issue name line, so that it formats properly.
Should the first Tx in the workaround be TX, or is the lowercase x part of the command's name?

Reply:
Should the `BT_CTLR_TX_PWR` be ``:kconfig:`CONFIG_BT_CTLR_TX_POWER` ``?
The description would just be a rephrasing of the title, so any suggestions?
"Zephyr Write Tx Power Level" is how it is described in https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/reference/bluetooth/hci.txt#L805

Reply:
Ah, yes, seems you are correct on the config formatting.

For description, I'd bring in setting TX power from the workaround:
"Using :kconfig:`BT_CTRL_TX_PWR` does not set TX power."